src/send/SendConfirmation.tsx

```diff
@@ -82,6 +82,7 @@ export function useRecipientToSendTo(paramRecipient: Recipient) {
         // Setting the phone number explicitly so Typescript doesn't complain
         e164PhoneNumber: paramRecipient.e164PhoneNumber,
         address: recipientAddress ?? undefined,
+        recipientType: recipientAddress ? RecipientType.Address : RecipientType.PhoneNumber,
```

Comment:
this is inside a block entered when phone number is defined and address is not defined. shouldn't the recipientType be phone number?

Reply:
Yeah good call, changed to phone number